test/api/buffer/receivers/delete/001-create-ok/expected-response.json

```diff
@@ -1,6 +1,5 @@
 {
   "receiverId": "delete-receiver-a",
```

Comment:
Dávalo by smysl tady vyměnit nějaký ten `create` za initial snapshot? Jen aby bylo vidět, jak to funguje

Reply:
review necham na Honzu, len jedna poznamka k tomuto:
tu mozno (na jednom mieste) este ano, len to prosim nenahradzujte vsade, ... pouzit iny request na setup je IMHO ok a jednoduchsie

Reply:
To bych asi nedělal, dělal jsem to kvůli importu a ten je tady: https://github.com/keboola/keboola-as-code/pull/1001/commits/f9b503b3701ee2c49e132ae412e584f9d415a5ce Chtěl jsem zamergovat všechno co můžu, ale je fér to dělat až společně s testem. 🙂 Tak bych to nechal na ten druhý PR a tenhle zavřel.